test/source/tests/test_get.brs

```diff
@@ -134,3 +142,74 @@ End Function
 Function testCase_get_badPathDefault()
   return m.AssertEqual(m._.get({a:1}, invalid, "default"), "default")
 End Function
+
+Function testCase_get_simpleField()
+  source = CreateObject("roSGNode", "ContentNode")
+  source.id = "parentNode"
+  child1 = source.createChild("ContentNode")
+  child1.id = "abc"
+
+  value = m._.get(source, "id")
+  return m.AssertTrue(value = "parentNode")
+End Function
+
+Function testCase_get_simpleNode()
```

Comment:
This is the exact same test logic with a different name as testCase_get_simpleField? Maybe a copy/paste error?